joint_state_controller/include/joint_state_controller/joint_state_controller.hpp

```diff
@@ -68,6 +65,7 @@ class JointStateController : public controller_interface::ControllerInterface
 
 protected:
   std::vector<std::string> joint_names_;
+  std::unordered_map<std::string, std::unordered_map<std::string, double>> name_if_value_mapping_;
```

Comment:
Can you a short comment about this map?

Reply:
Added comments, I've also grouped the joint-state-related stuff and the dynamic joint state-reltaed stuff together